Scripts/SL-Helpers.lua

```diff
@@ -620,12 +590,9 @@ CalculateExScore = function(player)
 	for index, window in ipairs(TNS) do
 		local number = stats:GetTapNoteScores( "TapNoteScore_"..window )
 
-		-- The W0 window needs to be emulated in ITG mode.
-		-- We already track note offsets so use that to compute the W0 count.
-		-- TODO(teejusb): Explicitly track this value instead of computing the count here
-		-- as it will be useful to display the count during gameplay.
+		-- The W0 window is emulated in ITG mode. Grab the value from stage stats.
 		if window == "W1" and SL.Global.GameMode == "ITG" then
-			local W0_count = GetW0Count(player)
+			local W0_count = SL[pn].Stages.Stats[SL.Global.Stages.PlayedThisGame + 1].W0_count
```

Comment:
Does this also work for courses?

Reply:
<img width="836" alt="image" src="https://user-images.githubusercontent.com/5017202/158073299-644fe6fb-1d4e-4c3b-8a8b-c1e15dd9a977.png">
<img width="885" alt="image" src="https://user-images.githubusercontent.com/5017202/158073305-a1654f90-103d-4bcc-bca3-09e68023a8d5.png">

39372 total notes, 535 total holds, 47 total rolls, 105 total mines in the cours

```
((912*3.5)+
 (480*3)+
 (741*2)+
  113+
  28-
  105)/(39372*3.5+535+47)*100 = 4.44
```
Looks like it works fine to me!